docs/DeveloperGuide.md

```diff
@@ -257,18 +273,18 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* works in the housing management team for RC4 with several other co-workers
-* has a need to manage a significant number of residents in RC4
+* works in the housing management team for [**RC4**](#glossary) with several other co-workers
+* has a need to manage a significant number of residents in [**RC4**](#glossary)
```

Comment:
I think this was a point made in class, are we going to to hyperlink every references made to the glossary? or just the first instance used in the entire document?

Reply:
I'm inclined to hyperlink every reference, there's no harm and it's easy enough. Unless there are other considerations that I missed out